12-http/example3/main.go

```diff
@@ -18,20 +21,46 @@ type user struct {
 	Phone string
 }
 
-// users is a slice of users.
-var users []user
+var (
+	// mu protects users.
+	mu sync.RWMutex
+	// initialize users, so that the output JSON won't be null
+	// prior to adding users.
+	users = make([]user, 0)
+)
+
+// index page template
+var IdxTpl = template.Must(template.ParseFiles("template/index.html"))
+
+// file server for any future assets and other static files
```

Comment:
"represents" is generally a word used in type declaration doc comments. I'll use the word "is".